commands/build.ts

```diff
@@ -108,6 +117,21 @@ export const handler: Handler<Args> = async function handler(argv, { getWorkspac
 				],
 			});
 
+		bins.forEach(([dir, src], name) => {
```

Comment:
whats this additional build step for?

Reply:
I pulled this out for anything that's an executable `bin`. I've been going a bit back and forth on it. I'd like to not have `--packages=external`, but there were some small issues last time I tried (they may be resolved now, need to re-check)